src/extension/services/files.service.ts

```diff
@@ -0,0 +1,14 @@
+class filesService {
+	getModesOfUse(): Promise<any> {
+		return fetch(chrome.runtime.getURL('assets/json/modes-of-use.json'))
+			.then(response => response.json())
+			.then(data => {
+				// Faites quelque chose avec les données JSON, par exemple :
+				return data;
+			})
+			.catch(error => {
+				console.error('Erreur lors de la récupération du fichier JSON :', error);
```

Comment:
_My two cents_ : les messages d’erreur devraient être en anglais, aussi :)

Et en bonus, tu pourrais utiliser un _template literal_, ça paraîtrait plus sain :
```suggestion
				console.error(`Erreur lors de la récupération du fichier JSON : ${error}.`);
```